spec/services/resync_service_spec.rb

```diff
@@ -82,5 +82,58 @@
         expect(request).to have_been_requested
       end
     end
+
+    context "when the live edition has been removed" do
+      let(:explanation) { "explanation" }
+
+      before do
+        stub_any_publishing_api_unpublish
+        allow(PreviewAssetService).to receive(:call)
+        allow(PublishAssetService).to receive(:call)
```

Comment:
Do these stubs need to be here? not sure if they have an impact.

Reply:
I was getting an because I'd set up the factory incorrectly. Will remove them.